plugins/org.jkiss.dbeaver.ext.intersystems/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jkiss.dbeaver</groupId>
+    <artifactId>plugins</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+     <relativePath>../</relativePath>
+  </parent>
+  <artifactId>org.jkiss.dbeaver.ext.intersystems</artifactId>
+  <packaging>eclipse-plugin</packaging>
+  <dependencies>
+    <dependency>
+      <groupId>org.jkiss.dbeaver</groupId>
+      <artifactId>org.jkiss.dbeaver.model</artifactId>
+      <version>1.1.24-SNAPSHOT</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jkiss.dbeaver</groupId>
+      <artifactId>org.jkiss.dbeaver.ext.generic</artifactId>
+      <version>2.3.217-SNAPSHOT</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jkiss.dbeaver</groupId>
+      <artifactId>org.jkiss.dbeaver.model.jdbc</artifactId>
+      <version>1.0.24-SNAPSHOT</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+    	<groupId>org.jkiss.dbeaver</groupId>
+    	<artifactId>org.jkiss.dbeaver.model.sql</artifactId>
+    	<version>1.0.121-SNAPSHOT</version>
+    </dependency>
+  </dependencies>
```

Comment:
We don't put dependencies in pom like this, they should be specified in MANIFEST.MF like in other packages

Reply:
I did this in the recent commit